sycl/include/sycl/detail/backend_traits_level_zero.hpp

```diff
@@ -35,6 +36,11 @@ typedef struct _ze_image_handle_t *ze_image_handle_t;
 typedef struct _ze_kernel_handle_t *ze_kernel_handle_t;
 typedef struct _ze_module_handle_t *ze_module_handle_t;
 
+typedef struct _ze_queue_or_cmdlist {
+  void *ze_handle;
+  bool IsImmCmdList;
+} *ze_queue_or_cmdlist_handle_t;
```

Comment:
ze_queue_or_cmdlist_handle_t: some suggestions for name:

ze_queue_object_handle_t
ze_queue_stream_handle_t
ze_stream_handle_t
ze_submission_stream_t
ze_submission_object_t

Reply:
I even suggest to avoid "ze" prefix to avoid confusion with something provided by the Level-Zero spec

Reply:
@smaslov-intel : right, that would be better.